docs/_docs/includes.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: docs
+title: Includes
+permalink: /docs/includes/
+---
+
+The `include` tag allows you to include the content from another file stored in the `_includes` folder:
+
+```liquid
+{% raw %}{% include footer.html %}{% endraw %}
+```
+
+Jekyll will look for the referenced file (in this case, `footer.html`) in the `_includes` directory at the root of your source directory and insert its contents.
+
+### Including files relative to another file
+
+You can choose to include file fragments relative to the current file by using the `include_relative` tag:
+
+```liquid
+{% raw %}{% include_relative somedir/footer.html %}{% endraw %}
+```
+
+You won't need to place your included content within the `_includes` directory. Instead,
+the inclusion is specifically relative to the file where the tag is being used. For example,
+if `_posts/2014-09-03-my-file.markdown` uses the `include_relative` tag, the included file
+must be within the `_posts` directory or one of its subdirectories.
+
+Note that you cannot use the `../` syntax to specify an include location that refers to a higher-level directory.
+
+All the other capabilities of the `include` tag are available to the `include_relative` tag,
+such as variables.
+
+### Using variables names for the include file
+
+The name of the file you want to embed can be specified as a variable instead of an actual file name. For example, suppose you defined a variable in your page's frontmatter like this:
+
+```yaml
+---
+title: My page
+my_variable: footer_company_a.html
+---
+```
+
+You could then reference that variable in your include:
+
+```liquid
+{% raw %}{% include {{page.my_variable}} %}{% endraw %}
+```
+
+In this example, the include would insert the file `footer_company_a.html` from the `_includes/footer_company_a.html` directory.
+
+### Passing parameters to includes
+
+You can also pass parameters to an include. For example, suppose you have a file called `note.html` in your `_includes` folder that contains this formatting:
+
+```liquid
+{% raw %}<div markdown="span" class="alert alert-info" role="alert">
+<i class="fa fa-info-circle"></i> <b>Note:</b> {{include.content}}
+</div>{% endraw %}
+```
```

Comment:
> At some point, i'd like to create a style guide for the site that defines conventions like these.

Ben has already set of guidelines defined at https://github.com/benbalter/jekyll-style-guide. Add to it if not already defined..